GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSmoist_GridComp/pyMoist/pyMoist/field_types.py

```diff
@@ -0,0 +1,9 @@
+import gt4py.cartesian.gtscript as gtscript
+
+from ndsl.dsl.typing import Float
+from pyMoist.aer_activation_constants import n_modes
+from pyMoist.saturation.constants import TABLESIZE
+
+
+FloatField_NModes = gtscript.Field[gtscript.IJK, (Float, (n_modes))]
+FloatField_VaporSaturationTable = gtscript.Field[gtscript.K, (Float, (int(TABLESIZE)))]
```

Comment:
do we use this for anything that is not the VaporSaturationTable? If so can we rename it? 
```suggestion
VaporSaturationTable = gtscript.Field[gtscript.K, (Float, (int(TABLESIZE)))]
```

Reply:
It seems like these sorts of things are going to be fairly common, so I am going to make a module that allows us to create these dynamically, at which point this entire file will be deleted